editor/src/components/canvas/ui-jsx-canvas-renderer/scene-root.tsx

```diff
@@ -152,6 +149,7 @@ function useGetValidTemplatePaths(
 interface SceneRootRendererProps {
   sceneElement: JSXElement
   filePath: string
+  validPaths: Array<InstancePath>
```

Comment:
is this a preparation to consolidating the data-utopia-valid-paths to the canvas root element, and removing it from the scene element in the DOM?

Reply:
Yep. We were always adding them there but never passing them into the scene, so the scene would wipe them.